packages/next-codemod/bin/upgrade.ts

```diff
@@ -30,9 +31,20 @@ async function loadHighestNPMVersionMatching(query: string) {
 }
 
 export async function runUpgrade(
-  revision: string | undefined,
+  revision: string,
   options: { verbose: boolean }
 ): Promise<void> {
+  console.log(chalk.bold(`Next.js upgrade codemode (version: ${revision})`))
```

Comment:
It's a logging to indicate it started not installing the deps. users can know it earlier otherwise they don't know the version/dist-tag they update to until the installing